internal/locale/locales/en-us.yaml

```diff
@@ -1690,6 +1690,8 @@ err_no_projectfile:
   other: No activestate.yaml file exists in the current working directory or its parent directories.
 err_project_not_found:
   other: Could not find project at [NOTICE]{{.V0}}[/RESET]
+err_projectfile_parse:
+  other: Unable to parse {{.V0}}. Please try checking out your project again.
```

Comment:
We should include the parsing error and remove the blurb about "checking out again". It failing to parse is a legitimate input error that the user is expected to address because the activestate.yaml is meant to be edited by the user.

Reply:
The parse error is shown, but I will remove the blurb about checking out the project again.

```
Script Output
─────────────
% build/state run build
 x Unable to parse /Users/test/cli/activestate.yaml
 x Project file `/Users/test/cli/activestate.yaml` could not be parsed, the parser produced the following error: yaml: line 3: did not find expected key
```